app/uk/gov/hmrc/plasticpackagingtax/returns/controllers/actions/CommonAuth.scala

```diff
@@ -31,6 +31,15 @@ trait CommonAuth {
       agentCode and confidenceLevel and nino and saUtr and dateOfBirth and agentInformation and groupIdentifier and
       credentialRole and mdtpInformation and itmpName and itmpDateOfBirth and itmpAddress and credentialStrength and loginTimes
 
+  protected def acceptableCredentialStrength: Predicate = {
+    val strongCredentials = CredentialStrength(CredentialStrength.strong)
+    // Agents are allowed to use weak credentials
+    // The order of this clause is important if we wish to preserve the MFA uplift of non agents.
```

Comment:
This is important; as we use programming by exception to catch the MFA uplift journey start